aws_lambda.py

```diff
@@ -34,22 +46,35 @@ def white_bold(s):
     )
 
 
-if __name__ == "__main__":
+def run(command):
+    event = {"server": os.getenv("SERVER", "http://localhost:8888/v1")}
+    context = {"sentry_sdk": sentry_sdk}
+    # Note! If the sentry_sdk was initialized with
+    # the AwsLambdaIntegration integration, it is now ready to automatically
+    # capture all and any unexpected exceptions.
+    # See https://docs.sentry.io/platforms/python/aws_lambda/
+    command(event, context)
+
+
+def main(*args):
     # Run the function specified in CLI arg.
     #
     # $ AUTH=user:pass python aws_lambda.py refresh_signature
     #
-    event = {"server": os.getenv("SERVER", "http://localhost:8888/v1")}
-    context = None
 
-    entrypoint = sys.argv[1]
-    if entrypoint == "help":
-        help()
-        sys.exit(0)
+    if not args or args[0] in ("help", "--help"):
+        help_()
+        return
+    entrypoint = args[0]
     try:
         mod = importlib.import_module(f"commands.{entrypoint}")
-        getattr(mod, entrypoint)(event, context)
+        command = getattr(mod, entrypoint)
     except (ImportError, ModuleNotFoundError):
-        print(f"Unknown function '{entrypoint}'")
-        help()
-        sys.exit(1)
+        print(f"Unknown function {entrypoint!r}", file=sys.stderr)
+        help_()
+        return 1
+    run(command)
+
+
+if __name__ == "__main__":
+    sys.exit(main(*sys.argv[1:]))
```

Comment:
All of the above is just reorganize it a little. The `if __name__ == '__main__'` is impossible to automation test so I made this line absolutely minimal.